server/routes/cookies/CookiesPolicyController.ts

```diff
@@ -38,6 +38,7 @@ export default class CookiesPolicyController {
           maxAge: 365 * 24 * 60 * 60 * 1000,
           sameSite: 'lax',
           secure: true,
+          httpOnly: true,
```

Comment:
I took the opportunity of making the cookie preferences cookie http only to tick off one of the low BSI recommendations.
Seems sensible to get this into prod before swathes of users start using the system and creating the cookie without the http only flag